lib/skylight/probes/elasticsearch.rb

```diff
@@ -0,0 +1,27 @@
+module Skylight
+  module Probes
+    module Elasticsearch
+      class Probe
+        def install
+          ::Elasticsearch::Transport::Transport::Base.class_eval do
+            alias perform_request_without_sk perform_request
+            def perform_request(method, path, *args, &block)
+              ActiveSupport::Notifications.instrument "request.elasticsearch",
```

Comment:
Wouldn't it make more sense to just call `Skylight.instrument` directly here, instead of going through a normalizer?